health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppFeatures.js

```
@@ -210,34 +302,52 @@
-              }
-            }}
+            onClick={() => handleConfirmCacheClear()}
           />,
         ]}
       />
+      {showPopUp && (
+        <PopUp
+          className={"boundaries-pop-module"}
+          type={"default"}
+          heading={t("SURE_TO_CLEAR_CACHE")}
+          children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)